README.md

```diff
@@ -15,7 +94,52 @@ have pre-slotted error codes for each library. The currently allocated error ran
 | [0, 0x0400) | aws-c-common |
 | [0x0400, 0x0800) | aws-c-io |
 | [0x0800, 0x1000) | aws-c-http |
-| [0x1000, 0x2000) |aws-c-amazon-flow | 
+| [0x1000, 0x2000) |aws-c-eventstream | 
 | [0x2000, 0x2800) | (reserved for future project) |
 
-Each library should begin its error codes at the beginning of its range and follow in sequence (don't skip codes).
+Each library should begin its error codes at the beginning of its range and follow in sequence (don't skip codes). Upon
+adding an AWS maintained library, an error code range must be approved and added to the above table.
+
+### Testing
+PRs without unit tests will not be acknowledged. A couple of tips:
```

Comment:
We should still acknowledge and request changes...

Reply:
by contract with the user, no, but to be nice and productive (probably).

We'll probably say something like, let me know when you've tested this and I'll have a look

Reply:
maybe the wording should be something like:

"We will not evaluate a PR without unit tests or if the unit tests aren't passing other than to ask you to please write unit tests."

Reply:
Even that feels a bit strong as an absolute. What if someone wants review of an API sketch before they implement it? What if it's a PR that just does formatting changes, or improves efficiency but doesn't change behavior and thus doesn't have any unit tests?

It's better to just say that we have a high bar for test coverage, and PRs fixing bugs or introducing new functionality need to have tests before they will be accepted.

Reply:
sure, will update